src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -15,11 +15,9 @@ import { FileEntryBlock, FilePairBlock, FileHeaderWithCheckbox, handleFileCheckb
 import { SelectedFilesController } from './SelectedFilesController';
 import { memoizedUrlParse } from './../../globals';
 import { expFxn, Schemas, typedefs, fileUtil } from './../../util';
+import { Item, ExperimentSet } from '../../util/typedefs';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['ExperimentSet' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4885443)